README.md

```diff
@@ -1,22 +1,61 @@
-## Mento SDK
+# Mento SDK
 
-This repo contains the source code of the software development kit for the Mento protocol. The repository is built with npm which is used for the compilation, testing and deployment of the sdk.
+The official Mento Protocol SDK for interacting with Multi-Collateral Mento smart contracts on the Celo network.
 
-## Getting Started
+# Sample Usage
 
-```bash
-# Get the latest code
-git clone git@github.com:mento-protocol/mento-sdk.git
+```javascript
+const provider = new providers.JsonRpcProvider(
+  'https://baklava-forno.celo-testnet.org'
+)
+const pKey = 'privateKey'
+const wallet = new Wallet(pKey, provider)
 
-# Change directory to the the newly cloned repo
-cd mento-sdk
+const mento = await Mento.create(wallet)
 
-# Install dev dependencies with yarn
-yarn
+console.log('available pairs: ', await mento.getTradeablePairs())
+/*
+        [
+      [
+        {
+          address: '0x62492A644A588FD904270BeD06ad52B9abfEA1aE',
+          symbol: 'cUSD'
+        },
+        {
+          address: '0xdDc9bE57f553fe75752D61606B94CBD7e0264eF8',
+          symbol: 'CELO'
+        }
+      ],
+      ...
+    ]
+    */
 
-# Build the sdk with yarn
-yarn build
+// swap 1 CELO for cUSD
+const one = 1
+const tokenIn = '0xdDc9bE57f553fe75752D61606B94CBD7e0264eF8' // CELO
+const tokenOut = '0x62492A644A588FD904270BeD06ad52B9abfEA1aE' // cUSD
+const amountIn = utils.parseUnits(one.toString(), 18)
+const amountOutMin = await mento.getAmountOut(
+  tokenIn,
+  tokenOut,
+  // 95% of amountIn to allow some slippage
+  utils.parseUnits((one * 0.95).toString(), 18)
```

Comment:
While in some sense this achieves the same result, I feel like it would be more natural to scale the return value of getAmountOut by 0.95 instead of the amountIn passed to it. Maybe call the result `expectedAmountOut` and then defined `minAmountOut = expectedAmountOut * 0.95`

Reply:
Yeah this was my first thought as well but BigNumber in ethers only supports integers, so I can't do `expectedAmountOut.mul(0.95)`

Reply:
chatted offline, changed this to `minAmountOut = expectedAmountOut.mul(95).div(100)`